### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by justcallmekoko**
Fix M5Stick-C Plus Menu display

### CODE REVIEW

File: esp32_marauder/configs.h
Comment (Artfuldodger808) [inline esp32_marauder/configs.h:1]: Hi mate, sorry to bother you but what does (All checks have failed
1 failing check) mean? Sorry if that's just plain annoying feel free to ignore me ✌

Comment (Artfuldodger808) [reply esp32_marauder/configs.h:1]: Oh , I asked gpt3.5 and it explained it pretty well lol, thanks anyway I'll probably just delete these comments lol

**Comment by Artfuldodger808** [comment]
I'm confused if it's resolved, I'm no expert on here but I am trying to learn. Thank you very much for doing anything related to this in general anyways , I appreciate it mate ✌, have a good one

**Comment by justcallmekoko** [comment]
@Artfuldodger808 What is this in reference to?

**Comment by Artfuldodger808** [comment]
@justcallmekoko
[Fix M5Stick-C Plus display]
[ea09719]
All checks have failed
1 failing check
@github-actions
Build and Push / Build Marauder Binaries

**Pull request merged**